web/src/lib/components/elements/dropdown.svelte

```diff
@@ -1,12 +1,12 @@
 <script lang="ts">
-  import SwapVertical from 'svelte-material-icons/SwapVertical.svelte';
   import Check from 'svelte-material-icons/Check.svelte';
   import LinkButton from './buttons/link-button.svelte';
   import { clickOutside } from '$lib/utils/click-outside';
   import { fly } from 'svelte/transition';
 
   export let options: string[] = [];
   export let value = options[0];
+  export let icons: any[] = undefined;
```

Comment:
Can you change the options input to be something like this instead?

```
interface DropdownOption {
  value: string;
  icon?: typeof LinkButton;
}

export let options: DropdownOption[] = [];
```

Reply:
👍 
Added interface for DropdownOption.